playbooks/luna-devel.yml

```diff
@@ -3,12 +3,11 @@
   vars:
     foreman_installer_options:
       - "--katello-devel-github-username '{{ katello_devel_github_username }}'"
-      - "--katello-devel-enable-ostree=true"
-      - "--katello-devel-extra-plugins theforeman/foreman_remote_execution"
-      - "--katello-devel-extra-plugins theforeman/foreman_discovery"
-      - "--katello-devel-extra-plugins theforeman/foreman_ansible"
-      - "--katello-devel-extra-plugins theforeman/foreman-tasks"
-      - "--katello-devel-extra-plugins theforeman/foreman_bootdisk"
-      - "--katello-devel-extra-plugins theforeman/foreman_openscap"
-      - "--katello-devel-extra-plugins theforeman/foreman_templates"
-      - "--katello-devel-extra-plugins theforeman/foreman_hooks"
+      - '--katello-devel-extra-plugins theforeman/foreman_remote_execution'
```

Comment:
Why are the different quotes needed?

Reply:
blame my editor! it works fine with single quotes too, so i did not mind it. If you insist i can change it back :)

Reply:
I see in github action, it says syntax error